### DISCUSSION TITLE
Why did the number of my TApp tokens increase?

### DISCUSSION THREAD

**Discussion by lxm218** [tearust/teaproject#66] [Q&A] [answered]
I bought 10 token for ID 3 and 4, 20 tokens for ID 1

![image](https://user-images.githubusercontent.com/12605218/132892018-bff26644-2a3e-468b-9422-f98dd776405e.png)

**Comment by kevingzhang** [comment, marked as answer]
This is the magic of the bonding curve. The TEA project uses what's called an augmented bonding curve. It supports both "consume" and "expense" actions. Your token supply will increase when there are consume actions for the TApp and decrease when there are expense actions coming from the TApp.

#### Consume Action
When a consumer spends (pays for usage) of a TApp, the TEA they spend is paid into this TApp's operational account. Since the consumer has used the service in exchange for TEA, their transaction is completed. 

The TEA they spend on the TApp is used to buy its TApp token. These newly minted tokens will then be distributed to all token holders, and that is how your token balance increases. Because the person who paid to use the TApp has already received the service already, they won't receive any newly minted tokens for their transaction. 

Investors who own a TApp's token are stakeholders in that app. They're like stock holders own some percentage of the project and receive a dividend every time the TApp is used.

#### Expense Action
On the other hand, when the TApp pays the hosting fee to miners, this is called an "expense". This is the opposite operation of "consume": the TApp will sell the TApp token for TEA to pay for hosting services. The token comes from all TApp token holders, which will cause their token balance to decrease.

There is a chance that a TApp is not popular and cannot earn enough money from consumers to support its payment to miners. In this case, the TApp's token price and total supply will drop to zero. This would be the death of the TApp. Of course, investors lose all of their investment in this scenario so choose your TApp tokens wisely.

#### Daily Consumes in Epoch 3
In epoch 3, the consumer is a non-playing character (NPC) who we'll call the "big boss". The big boss comes around every day to choose the best performing TApp and pays a large amount of TEA to consume. This is called "tipping". The performance is measured by the daily rate of increase in YouTube views. The TApp with the biggest increase will get the tip.

So the best strategy here is to invest in the YouTube video you believe will have the greatest daily increase in views.
